canvas_account_admin_tools/templates/canvas_account_admin_tools/dashboard_account.html

```diff
@@ -4,68 +4,75 @@
 
 {% block stylesheets %}
 <link href="{% static 'canvas_account_admin_tools/css/dashboard.css' %}" rel="stylesheet" />
+  <style>
+    a.btn {
+        width: 20em;
+        margin-top: 1em;
+    }
+  </style>
 {% endblock stylesheets %}
 
 {% block content %}
-<header>
-    <div class="row lti-header">
-        <h1>Administration Tasks</h1>
-    </div>
-</header>
 
-<main>
+    <header>
+      <nav class="ng-scope">
+        <h3>Admin Console</h3>
+      </nav>
+    </header>
+
     {% if has_manage_courses %}
```

Comment:
It looks like all these perms are going away in another ticket that was added today.